system-tests/cli/cli.test.js

```diff
@@ -114,4 +114,46 @@ describe('CLI', () => {
 			);
 		});
 	});
+
+	it('--noglob-enabled-right', () => {
```

Comment:
Thanks for PR, it really useful feature. Could you please rename test cases without additional `-` and make more informative description. For example
```
describe('--no-glob', () => {
     it('enabled. file exists');
     it('enabled. file does not exist');
     it('disabled.  file exists');
     it('disabled. file does not exist');
})
```
```